public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -426,40 +428,40 @@ code-example(format).
 +makeExample('dependency-injection/ts/app/providers.component.ts','provider-10-ctor')(format='.')
 :marked
   Be prepared for a null logger. If we don't register one somewhere up the line,
-  the injector will inject `null`. We have a method that logs. 
+  the injector will inject `null`. We have a method that logs.
   What can we do to avoid a null reference exception?
-  
+
   We could substitute a *do-nothing* logger stub so that calling methods continue to work:
 +makeExample('dependency-injection/ts/app/providers.component.ts','provider-10-logger')(format='.')
 :marked
   Obviously we'd take a more sophisticated approach if the logger were optional
-  in multiple locations. 
-  
-  But enough about optional loggers. In our sample application, the `Logger` is required. 
-  We must register a `Logger` with the application injector using *providers*
+  in multiple locations.
+
+  But enough about optional loggers. In our sample application, the `Logger` is required.
+  We must register a `Logger` with the application injector using *providers*,
   as we learn in the next section.
 
 <a id="providers"></a>
 .l-main-section
 :marked
-  ## Injector Providers
-  
+  ## Injector providers
+
   A provider *provides* the concrete, runtime version of a dependency value.
-  The injector relies on **providers** to create instances of the services 
-  that it injects into components and other services.
+  The injector relies on **providers** to create instances of the services
+  that the injector injects into components and other services.
+
+  We must register a service *provider* with the injector, or it won't know how to create the service.
 
-  We must register a service *provider* with the injector or it won't know how to create the service. 
-  
   Earlier we registered the `Logger` service in the `providers` array of the metadata for the `AppComponent` like this:
 +makeExample('dependency-injection/ts/app/providers.component.ts','providers-logger')
 :marked
-  The `providers` array appears to hold a service class. 
-  In reality it holds an instance the [Provider](../api/core/Provider-class.html) class that can create that service.
- 
+  The `providers` array appears to hold a service class.
+  In reality it holds an instance of the [Provider](../api/core/Provider-class.html) class that can create that service.
```

Comment:
Looking around in template-syntax.jade, it appears that `../api/` -> `/docs/ts/latest/api/`.